weblate/formats/ttkit.py

```diff
@@ -670,7 +670,7 @@ def target(self):
 
 
 class PoFormat(TTKitFormat):
-    name = _('Gettext PO file')
+    name = _('gettext PO file')
```

Comment:
These are labels for the file format, all are starting with uppercase, so I'd rather keep this consistent.

Reply:
Hmmm, I figured it was easier to always use it like upstream gettext uses it.
Is it a solution to go "GNU gettext"?
Also, I've seen "i18next", which isn't "I18next"

Reply:
True, we should be consistent with i18next as well. However even upstream is not consistent on capitalization here when used as first word in a sentence, see https://www.i18next.com/. But in most case they also use lowercase...